onnx/backend/test/case/node/lppool.py

```diff
@@ -0,0 +1,247 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ..base import Base
+from . import expect
+from .pool_op_common import get_output_shape, get_pad_shape, pool
+
+
+class LpPool(Base):
+    @staticmethod
+    def export_lppool_1d_default() -> None:
+        """
+        input_shape: [1, 3, 32]
+        output_shape: [1, 3, 31]
+        """
+        p = 3
```

Comment:
I see 3 and 4 used a lot for p. May be worth changing some of them to p=1.

Reply:
p=1 will be proportionally AvgPool. I am flexible there though.

Reply:
Just to clarify: I think p=1 differs from AveragePool in two ways: first, it uses the absolute value and second, it computes a sum, not average.

Reply:
the absolute value is what we shall discuss - I do not see pytorch's implementation does that. there is an issue which we shall solve in ONNX and then in Ort. https://github.com/microsoft/onnxruntime/issues/6302. With that issue soloved, p=1 is close to AvgPool as Pytorch document stated:
At p = 
∞
∞, one gets Max Pooling

At p = 1, one gets Sum Pooling (which is proportional to average pooling)